src/classes/ERR_ExceptionHandler.cls

```diff
@@ -98,13 +98,24 @@ public with sharing class ERR_ExceptionHandler {
         return false;
     }
 
+    @testVisible
+    private static AFFL_MultiRecordTypeMapper affilMapper {
+        get {
+            if(affilMapper == null) {
+                affilMapper = new AFFL_MultiRecordTypeMapper();
+            }
+
+            return affilMapper;
+        }
+        set;
+    }
+
     public static void handleAfflNullRecordTypeException(Affiliation__c affl, List<String> validRecordTypes) {
```

Comment:
Noting here that validRecordTypes is currently including ALL record types that are present in the mappings (not field specific). So I can still put a business organization in the Prim Education Inst field on contact. This PR doesn't fix or further break this issue and so we can address the bug later on.

Reply:
Agreed. This functionality is not appropriately enforced.